WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -1132,15 +1145,15 @@ nick_the_twister = {
 	chance = {
 		factor = 1
 		modifier = {
-			factor = 1.5
+			factor = 1
 			martial = 17
 		}
 		modifier = {
-			factor = 1.5
+			factor = 1
 			trait = impaler
 		}
 		modifier = {
-			factor = 2
+			factor = 1
```

Comment:
This particular name is far too common, and it doesn't make sense in the time period.